JungwooJang119/SylvieConstellation
# Beginning stages of image display for dialogue manager

To test:
- Go to Assets/Scenes/Avery and run Test
- There are two NPCs in the scene. Go to the one farther to the right because that NPC's dialogue scripts (namely LoversNPC.yarn) are so far the only ones I've modified thus far to test any of this out.
- Right now the two main files I've modified were NPCDialogue.cs (has the yarn command) and CharacterImageView.cs (has the image to display (or should display))
- Currently the error is: `Can't call command show_image, because CharacterImageView doesn't have the correct component`. Unfortunately this is where I'm stuck.
